### PR TITLE
[KissAsian] fix and small refactor

### LINKED ISSUE #3566: [Presence | Bug] Kissasian
**Description of the Bug**
Kissasian status isnt showing up due to a domain change.

**Steps to Reproduce**
N/A

**Expected Behavior**
Kissasian to be shown as my discord status

**Screenshots**
N/A

**Environment**
- OS: Linux
- Browser: Chrome
- Version:  89.0.4389.116 (Official Build) (64-bit)

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by mlunax**
Resolve #3566

### CODE REVIEW

File: websites/K/KissAsian/presence.ts [hunk 1]
```diff
@@ -15,61 +15,72 @@ function getTimestamps(
   videoTime: number,
   videoDuration: number
 ): Array<number> {
-  var startTime = Date.now();
-  var endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
+  const startTime = Date.now(),
+   endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
   return [Math.floor(startTime / 1000), endTime];
 }
```
Comment (ririxidev) [inline websites/K/KissAsian/presence.ts:21]: yeah use `getTimestamps` function that is inside `Presence` class.

File: websites/K/KissAsian/presence.ts [hunk 2]
```diff
@@ -15,61 +15,72 @@ function getTimestamps(
...
+    } else if (episodeSelection) {
+      episodeTitle = document
+        .querySelector("div#player-content > header > h1").textContent;
+        // .textContent.replace("information", "")
+        // .replace("Drama", "");
+      dramaTitle = document
+      .querySelector(".meta-cat > a").textContent;
 
-      timestamps = getTimestamps(Math.floor(currentTime), Math.floor(duration));
+      const timestamps = getTimestamps(Math.floor(currentTime), Math.floor(duration));
```
Comment (ririxidev) [inline websites/K/KissAsian/presence.ts:83]: like above

File: websites/K/KissAsian/presence.ts [hunk 3]
```diff
@@ -78,8 +89,8 @@ presence.on("UpdateData", async () => {
         presenceData.startTimestamp = timestamps[0];
```
Comment (ririxidev) [inline websites/K/KissAsian/presence.ts:89]: don't need to set startTimestamp

**Review by ririxidev — CHANGES_REQUESTED** [review]
also move `startTimestamp` to `presenceData` object

File: websites/K/KissAsian/dist/metadata.json [hunk 4]
```diff
@@ -3,13 +3,17 @@
...
+    "id": "251801168689364992"
+  }],
   "service": "KissAsian",
   "description": {
     "en": "Watch drama online in high quality. Free download high quality drama. Various formats from 240p to 720p HD (or even 1080p). HTML5 available for mobile devices.",
     "nl": "Bekijk drama online in hoge kwaliteit. Gratis download van hoge kwaliteit drama. Verschillende formaten van 240p tot 720p HD (of zelfs 1080p). HTML5 beschikbaar voor mobiele apparaten."
   },
-  "url": "kissasian.sh",
-  "version": "2.2.7",
+  "url": "kissasian.la",
```
Comment (Bas950) [inline websites/K/KissAsian/dist/metadata.json:15]:
```suggestion
  "url": ["kissasian.la", "kissasian.li"],
```

**Review by Bas950 — CHANGES_REQUESTED** [review]
And changes Ririxi mentioned

**Review by ririxidev — APPROVED** [review]

File: websites/K/KissAsian/presence.ts [hunk 5]
```diff
@@ -1,85 +1,90 @@
...
 presence.on("UpdateData", async () => {
   const presenceData: PresenceData = {
-    largeImageKey: "ka"
-  };
-
-  if (document.location.hostname == "kissasian.sh") {
+      largeImageKey: "ka"
+    },
+    episodeSelection = document.querySelector("div#all-episodes");
+  if (document.location.hostname == "kissasian.la") {
```
Comment (Bas950) [inline websites/K/KissAsian/presence.ts:41]: Remove this "if" case or make it work for both domains.

**Review by Bas950 — CHANGES_REQUESTED** [review]
And changes Ririxi mentioned

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**